### PR TITLE
[ spi_host, prj ] Update SPI_HOST checklist

### PR DESCRIPTION
**Pull request by martin-lueker**
- Indicate remaining PR's blocking D2
- Mark all V1 and V2 items as IN PROGRESS
- Mark D2S, V2S sections as N/A

Signed-off-by: Martin Lueker-Boden <martin.lueker-boden@wdc.com>

### CODE REVIEW

**Comment by martin-lueker** [comment]
Can we get a review on this?

File: hw/ip/spi_host/doc/checklist.md
Comment (rasmus-madsen) [inline hw/ip/spi_host/doc/checklist.md:178]: not started

File: hw/ip/spi_host/doc/checklist.md
Comment (rasmus-madsen) [inline hw/ip/spi_host/doc/checklist.md:180]: not started

File: hw/ip/spi_host/doc/checklist.md
Comment (rasmus-madsen) [inline hw/ip/spi_host/doc/checklist.md:185]: not started

File: hw/ip/spi_host/doc/checklist.md
Comment (rasmus-madsen) [inline hw/ip/spi_host/doc/checklist.md:188]: not started

File: hw/ip/spi_host/doc/checklist.md
Comment (rasmus-madsen) [inline hw/ip/spi_host/doc/checklist.md:186]: not started

File: hw/ip/spi_host/doc/checklist.md
Comment (rasmus-madsen) [inline hw/ip/spi_host/doc/checklist.md:187]: not started

File: hw/ip/spi_host/doc/checklist.md
Comment (rasmus-madsen) [inline hw/ip/spi_host/doc/checklist.md:192]: Complete

File: hw/ip/spi_host/doc/checklist.md
Comment (rasmus-madsen) [inline hw/ip/spi_host/doc/checklist.md:193]: not started

File: hw/ip/spi_host/doc/checklist.md
Comment (rasmus-madsen) [inline hw/ip/spi_host/doc/checklist.md:194]: not started

File: hw/ip/spi_host/doc/checklist.md
Comment (rasmus-madsen) [inline hw/ip/spi_host/doc/checklist.md:195]: not started

File: hw/ip/spi_host/doc/checklist.md
Comment (rasmus-madsen) [inline hw/ip/spi_host/doc/checklist.md:181]: complete

File: hw/ip/spi_host/doc/checklist.md
Comment (rasmus-madsen) [inline hw/ip/spi_host/doc/checklist.md:177]: only in progress in terms of V1 - not started for V2 as this is one of the very last tasks to complete

File: hw/ip/spi_host/doc/checklist.md
Comment (rasmus-madsen) [inline hw/ip/spi_host/doc/checklist.md:145]: I don't think this can be done before the V1 review

**Review by rasmus-madsen — DISMISSED** [review]
@martin-lueker I have added my comments
I am not sure what the intention of this PR is.
IMO we don't update the checklist to show "live" status.
but use it as a checklist in the reviews to check everything is complete.
in either case, I think this is overestimating the status of the DV

**Comment by martin-lueker** [comment]
> @martin-lueker I have added my comments I am not sure what the intention of this PR is. IMO we don't update the checklist to show "live" status. but use it as a checklist in the reviews to check everything is complete. in either case, I think this is overestimating the status of the DV

Thanks for your feedback.  I had been hoping to use it as a tool for Anka to track progress to D2. (TBH, though it was usefull just now for reminding myself which PR's needed attention, but I suppose I don't need to hound for reviewers on that). 
I'll wait to ping everyone again until #9091 is merged.

**Comment by rswarbrick** [comment]
I'm not sure there's project-wide agreement on the question of when to update the checklist. From my point of view, updates to reflect the current status make perfect sense.

**Comment by martin-lueker** [comment]
Hi everyone,
  With the last of the RTL & Doc PRs merged, this IP is now ready for D2.  Could someone please take a look at the checklist?

File: hw/ip/spi_host/doc/checklist.md
Comment (msfschaffner) [inline hw/ip/spi_host/doc/checklist.md:57]: If there are no CDCs in this block, should this be set to N/A?

File: hw/ip/spi_host/doc/checklist.md
Comment (msfschaffner) [inline hw/ip/spi_host/doc/checklist.md:52]: I did a quick search. There are still a bunch of TODOs in DV code - but that is OK.
On the design side I found this:
https://github.com/lowRISC/opentitan/blob/4c3ae9eaf131ebe2fb5e7f25f6430ca5bc15b3b5/hw/ip/spi_host/rtl/spi_host.sv#L9

Is this still relevant? Or can it be removed?

**Review by msfschaffner — COMMENTED** [review]
Overall LGTM, with two comments.

File: hw/ip/spi_host/doc/checklist.md
Comment (martin-lueker) [reply hw/ip/spi_host/doc/checklist.md:52]: Ah, yes, that is old. It as been implemented.  It can be removed.

File: hw/ip/spi_host/doc/checklist.md
Comment (martin-lueker) [reply hw/ip/spi_host/doc/checklist.md:57]: Yes, I will set this to N/A

File: hw/ip/spi_host/doc/checklist.md
Comment (martin-lueker) [reply hw/ip/spi_host/doc/checklist.md]: Done.

File: hw/ip/spi_host/doc/checklist.md
Comment (martin-lueker) [reply hw/ip/spi_host/doc/checklist.md]: Done.

**Review by msfschaffner — APPROVED** [review]

**Comment by martin-lueker** [comment]
Hi @rasmus-madsen: This is waiting for your release (as you requested changes).  Do you approve?

**Pull request merged**
